opefeature-web-provider/src/provider.ts

```diff
@@ -0,0 +1,148 @@
+import {
```

Comment:
Presumably this should be part of a PR here https://github.com/open-feature/js-sdk-contrib/tree/main/libs/providers